app/services/nomis/elite2/prisoner_offender_manager.rb

```diff
@@ -0,0 +1,25 @@
+# Note this is currently deserializing keyworker
+module Nomis
+  module Elite2
+    class PrisonerOffenderManager
```

Comment:
I think this is PrisonOffenderManager rather than PrisonerOffenderManager

Reply:
Good catch @rossjones - will change